src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The href attribute is linking to a '.scss' file. Typically, SASS files need to be compiled into CSS before they can be used in an HTML document. Ensure that you are linking to a compiled CSS file instead.